Cdn_TotalCdn_Api.php

```diff
@@ -258,29 +259,50 @@ public function get_user() {
 	 *
 	 * @since x.x.x
 	 *
-	 * @param int $id The pull zone ID.
-	 *
 	 * @link https://docs.bunny.net/reference/pullzonepublic_index2
 	 *
+	 * @param int $id The pull zone ID.
+	 *
 	 * @return array|WP_Error API response or error object.
 	 */
 	public function get_pull_zone( $id ) {
 		$this->api_type = 'account';
+		$id             = empty( $id ) ? $this->pull_zone_id : $id;
 
 		return $this->wp_remote_get(
 			\esc_url( $this->api_base_url . '/pullzone/' . $id )
 		);
 	}
 
 	/**
-	 * Adds a new pull zone.
+	 * Gets the details of a specific pull zone from the provider.
 	 *
 	 * @since x.x.x
 	 *
-	 * @param array $data Data for the new pull zone.
+	 * @link https://cdn-api-dev-joec.boldgrid.com/docs/api#/operations/pullzone.get_from_provider
+	 *
+	 * @param int $id The pull zone ID.
+	 *
+	 * @return array|WP_Error API response or error object.
+	 */
+	public function get_pull_zone_from_provider( $id ) {
```

Comment:
Make the `$id` argument optional -- `?int $id = null`.
Also, update the comment to show optional and type "?int".